glib/src/property.rs

```diff
@@ -50,6 +54,12 @@ impl<T: Property> Property for Rc<T> {
 impl<T: Property> Property for Arc<T> {
     type Value = T::Value;
 }
+impl<T: IsA<Object> + Property> Property for WeakRef<T> {
+    type Value = T::Value;
+}
+impl<T: IsA<Object> + Property> Property for SendWeakRef<T> {
+    type Value = T::Value;
```

Comment:
`T` is a gobject, so `Property::Value` is `T` itself.  You should be able to just put `Value = T` in there.
Then, the bound can just be `T: IsA<Object>`

Reply:
It doesn't seem like I can get it to just `T: IsA<Object>`, but I could get it to `T: IsA<Object> + HasParamSpec`, should I do that?

Reply:
Yes, that's correct, `HasParamSpec` is needed